src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java

```diff
@@ -1,203 +1,39 @@
 package com.google.checkstyle.test.chapter4formatting.rule44columnlimit; // ok
 
-import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.*; // ok
+import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.PackageStatementTest; // ok
 
 final class InputColumnLimit {
   // Long line
   // ----------------------------------------------------------------------------------------------------
   // violation above 'Line is longer than 100 characters (found 105).'
-  // Contains a tab ->    <-
-  // Contains trailing whitespace ->
 
-  // Name format tests
-  //
-  /** Invalid format * */
-  public static final int badConstant = 2;
-
-  /** Valid format * */
-  public static final int MAX_ROWS = 2;
-
-  /** Invalid format * */
-  private static int badStatic = 2;
-
-  /** Valid format * */
-  private static int sNumCreated = 0;
-
-  /** Invalid format * */
-  private int badMember = 2;
-
-  /** Valid format * */
-  private int mNumCreated1 = 0;
-
-  /** Valid format * */
-  protected int mNumCreated2 = 0;
-
-  /** commas are wrong * */
-  private int[] mInts =
+  private int[] testing =
       new int[] {
         1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27
         // violation above 'Line is longer than 100 characters (found 105).'
       };
 
-  //
-  // Accessor tests
-  //
-  /** should be private * */
-  public static int sTest1;
-
-  /** should be private * */
-  protected static int sTest3;
-
-  /** should be private * */
-  static int sTest2;
-
-  /** should be private * */
-  int mTest1;
-
-  /** should be private * */
-  public int mTest2;
-
-  //
-  // Parameter name format tests
-  //
-
   /**
-   * @return hack
+   * Some javadoc.
+   *
    * @param badFormat1 bad format
    * @param badFormat2 bad format
    * @param badFormat3 bad format
+   * @return hack
    * @throws java.lang.Exception abc
    */
   int test1(int badFormat1, int badFormat2, final int badFormat3) throws java.lang.Exception {
     return 0;
   }
```

Comment:
kept this method on purpose to show that violation is given only on code which is longer than column limit 100

Reply:
had to re-arrange block tags to make them in proper order